lua/boo/init.lua

```diff
@@ -32,20 +37,35 @@ function M.boo()
 		})
 	end
 	if config.close_on_mouse_move then
-		vim.api.nvim_create_autocmd({ 'CursorMoved', 'CursorMovedI' }, {
+		vim.api.nvim_create_autocmd({ 'CursorMoved', 'CursorMovedI', 'InsertEnter' }, {
```

Comment:
Closing boo when entering insert mode could be useful. But this should have its own setting and not be part of `close_on_mouse_move`.

Reply:
it should be in `close on mouse move` because we do move the cursor when entering insert mode and yeah adding more settings is not feasible